src/ReleaseHistory.md

```diff
@@ -2,6 +2,7 @@
 
 ## **v2.0.0-rc3** (Unreleased)
 * BUGFIX: Eliminate `BA3003.EnableStackProtector` false positives when the target is statically linked. [744](https://github.com/microsoft/binskim/pull/744)
+* BUGFIX: Fix over-aggressive parsing of DWARF compilation units even when all related rules are disabled. [774](https://github.com/microsoft/binskim/pull/774)
```

Comment:
>aggressive [](http://example.com/codeflow?start=19&length=10)

I note that we don't actually have a test that ensures we don't get into this data. Should we create one?

A simple test that initializes an Elf binary and then ensures no values are populated may be sufficient.